python/Ganga/GPIDev/Lib/File/File.py

```diff
@@ -296,13 +300,30 @@ def isExecutable(self):
         file are checked"""
         return self.executable or is_executable(expandfilename(self.name))
 
+    def getAssociatedFiles(self):
+        if os.path.isfile(os.path.join(self.path(), 'associated_files.xml')):
+            fobj = open(os.path.join(self.path(), 'associated_files.xml'), "r")
```

Comment:
I forget the naming but isn't this safer as `with open( ... ) :` rather than opening and closing the file handler?

Reply:
good point.